src/gtfs/mod.rs

```diff
@@ -31,10 +31,21 @@ use anyhow::{anyhow, Context};
 use chrono_tz::Tz;
 use derivative::Derivative;
 use serde::{Deserialize, Serialize};
-use std::{collections::BTreeMap, fmt, path::Path};
+use std::{
+    collections::{BTreeMap, HashMap},
+    fmt,
+    path::Path,
+};
+
 use tracing::info;
 use typed_index_collection::CollectionWithId;
 
+#[cfg(feature = "gtfs")]
```

Comment:
I would have made them like this because these functions are really specific to parsing.

```suggestion
#[cfg(all(feature = "gtfs", feature = "parser"))]
```